src/scenes/ActivityLogRouter/ActivityLogView.tsx

```diff
@@ -1,22 +1,300 @@
-import React, { useCallback, useMemo, useState } from 'react';
+import React, { Fragment, useCallback, useMemo, useState } from 'react';
 
+import { Box, Grid, Typography, useTheme } from '@mui/material';
 import { Button } from '@terraware/web-components';
 
 import Page from 'src/components/Page';
 import PageHeaderProjectFilter from 'src/components/PageHeader/PageHeaderProjectFilter';
+import Card from 'src/components/common/Card';
 import { useLocalization } from 'src/providers';
 import { useParticipantData } from 'src/providers/Participant/ParticipantContext';
+import { Activity } from 'src/types/Activity';
+import { SearchNodePayload } from 'src/types/Search';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+
+import ActivityLogMapSplitView from './ActivityLogMapSplitView';
+import ActivityStatusBadge from './ActivityStatusBadge';
+import DateRange from './FilterDateRange';
+
+const MOCK_ACTIVITIES: Activity[] = [
```

Comment:
Mock activities will be removed soon in a future PR when service, redux, etc. is added for Activities.